src/OrganisationComp.js

```diff
@@ -1,15 +1,17 @@
 import React, { Component } from 'react';
 import './App.css';
+import Details from './Details'
 
 
 
 class OrganisationList extends Component {
-  renderOrganisation= () => {
-    const orgData=this.state.orgData;
-      return orgData.map(function renderData(detailsOfServiceProviders) {
+  renderOrganisations= () => {
+    const orgData=this.props.orgData;
+    //loops through list
+      return orgData.map(function renderData(detailsOfServiceProviders, index) {
```

Comment:
Just be careful with spacing, return should be on the same indentation as the const above, otherwise it looks like it is nested